### ISSUE TITLE
Clear-Host doesn't clear the PowerShell Output window in VSCode 1.8.1

### ISSUE THREAD

**Issue by pcgeek86** [PowerShell/vscode-powershell#432]
**Please fill in these details so that we can help you!**

### System Details

- Operating system name and version: Mac OS X Sierra
- VS Code version: 1.8.1
- PowerShell extension version: 0.8.0
- Output from `$PSVersionTable`:

```
Name                           Value
----                           -----
PSVersion                      6.0.0-alpha
PSEdition                      Core
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   3.0.0.0
GitCommitId                    v6.0.0-alpha.14
CLRVersion
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Issue Description

I am experiencing a problem with Clear-Host. It doesn't clear the PowerShell Output window in Visual Studio Code.

**Comment by General-Gouda** [comment]
To piggy-back on this issue I am finding similar behavior with the new PS Integrated Console. When Clear-Host is activated the console simply jumps down to a point where you cannot see any of the output above it but if you scroll up (using a mouse wheel for example) it reveals that the code above it is still there. 

System Details:

-     Operating system name and version: Windows 10 Anniversary Edition
-     PowerShell extension version: PS Extension Version: 

VSCode details:
Version 1.10.2
Commit 8076a19fdcab7e1fc1707952d652f0bb6c6db331
Date 2017-03-08T14:02:52.799Z
Shell 1.4.6
Renderer 53.0.2785.143
Node 6.5.0

$PSVersionTable output:
```
Name                           Value
----                           -----
PSVersion                      5.1.14393.953
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.953
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by daviwil** [comment]
I'll fix this as part of issue #551.  Thanks!
